FWCore/Framework/src/SubProcess.cc

```diff
@@ -327,20 +332,21 @@ namespace edm {
         return;
       }
     }
-    process(ep);
+    processAsync(std::move(iHolder),ep);
     /* END relevant bits from OutputModule::doEvent */
   }
 
   void
-  SubProcess::process(EventPrincipal const& principal) {
+  SubProcess::processAsync(WaitingTaskHolder iHolder,
+                           EventPrincipal const& principal) {
     EventAuxiliary aux(principal.aux());
     aux.setProcessHistoryID(principal.processHistoryID());
-
+    
```

Comment:
Is it possible to have your editor removing spurious whitespaces like this?  If not, it's just a few extra bytes....